CONTRIBUTING.rst

```diff
@@ -192,25 +202,24 @@ Supported Python Versions
 
 We support:
 
--  `Python 3.5`_
 -  `Python 3.6`_
 -  `Python 3.7`_
 -  `Python 3.8`_
+-  `Python 3.9`_
 
-.. _Python 3.5: https://docs.python.org/3.5/
 .. _Python 3.6: https://docs.python.org/3.6/
 .. _Python 3.7: https://docs.python.org/3.7/
 .. _Python 3.8: https://docs.python.org/3.8/
+.. _Python 3.9: https://docs.python.org/3.9/
```

Comment:
We don't yet support 3.9 (blocked by missing pyarrow wheels), thus we should manually override this change in `synth.py` for the time being

Reply:
No longer blocked in 2.7.0